docs/sources/installation/docker.md

```diff
@@ -69,10 +69,10 @@ Pass the plugins you want installed to Docker with the `GF_INSTALL_PLUGINS` envi
 
 ```bash
 docker run -d \
--p 3000:3000 \
---name=grafana \
--e "GF_INSTALL_PLUGINS=grafana-clock-panel,grafana-simple-json-datasource" \
-grafana/grafana
+  -p 3000:3000 \
+  --name=grafana \
+  -e "GF_INSTALL_PLUGINS=grafana-clock-panel,grafana-simple-json-datasource" \
+  grafana/grafana
 ```
 
 > If you need to specify the version of a plugin, then you can add it to the `GF_INSTALL_PLUGINS` environment variable. Otherwise, the latest will be assumed. For example: `-e "GF_INSTALL_PLUGINS=grafana-clock-panel 1.0.1,grafana-simple-json-datasource 1.3.5"`.
```

Comment:
```suggestion
> **Note:** If you need to specify the version of a plugin, then you can add it to the `GF_INSTALL_PLUGINS` environment variable. Otherwise, the latest is used. For example: `-e "GF_INSTALL_PLUGINS=grafana-clock-panel 1.0.1,grafana-simple-json-datasource 1.3.5"`.
```